helm/generated_examples/baremetal-affinity.yaml

```diff
@@ -172,3 +174,128 @@ spec:
         - name: local-storage
           hostPath:
             path: /mnt/disks
+---
+# Source: provisioner/templates/daemonset_windows.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: local-static-provisioner-win
+  namespace: default
+  labels:
+    helm.sh/chart: provisioner-2.5.0
+    app.kubernetes.io/name: provisioner
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/instance: local-static-provisioner
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: provisioner
+      app.kubernetes.io/instance: local-static-provisioner
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: provisioner
+        app.kubernetes.io/instance: local-static-provisioner
+      annotations:
+        checksum/config: 5a2fc47afffa2a982cff53af8b99da3694484be2a032fa8f6c723580d2d7fe19
```

Comment:
do you know what is this?

Reply:
the template in `helm/` generates this, more docs: https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments